docs/src/modules/redux/initRedux.js

```diff
@@ -23,9 +24,6 @@ function create(initialState) {
     // redux-logger needs this feature
     Object.hasOwnProperty('assign')
   ) {
-    // eslint-disable-next-line global-require
-    const createLogger = require('redux-logger').createLogger;
```

Comment:
Webpack is clever enough to remove `redux-logger` from prod bundles. One less global require :)